File: fizzbuzz2.rb
i = 1

def eval_mult(multiple, divider)
  multiple % divider == 0
end

while i <= 100
  if eval_mult(i,15)
    puts "FizzBuzz"
  elsif eval_mult(i,3)
    puts "Fizz"
  elsif eval_mult(i,5)
    puts "Buzz"
  else
    puts i
  end
  i += 1
end

File: fizzbuzz2.js
var multOfEval;
var logFizzBuzz;

multOfEval = function(multiple, divider){
  return multiple % divider === 0;
}

logFizzBuzz = function(x){
  if (multOfEval(x, 15)){
    console.log("FizzBuzz");
  } else if (multOfEval(x,3)){
    console.log("Fizz");
  } else if (multOfEval(x,5)){
    console.log("Buzz");
  } else{
    console.log(x);
  }

}

for(var i = 1; i <= 100; i ++){
  logFizzBuzz(i);
}